Src/FluentAssertions/Formatting/PredicateLambdaExpressionValueFormatter.cs

```diff
@@ -91,7 +100,9 @@ public override Expression Visit(Expression node)
                 return node;
             }
 
-            if (!HasLiftedOperator(node) && ExpressionIsConstant(node))
+            if (node is not (MemberInitExpression or NewExpression)
```

Comment:
🤔 That took me a couple of times before I was convinced I was reading this correctly.

Reply:
The logical expression `!(p or q)` ([De Morgan's Law](https://en.wikipedia.org/wiki/De_Morgan%27s_laws)), the readability of the code or that it programmatically works?

Reply:
The readability